<?xml version="1.0" encoding="UTF-8" ?>
<testsuites name="vitest tests" tests="1" failures="1" errors="0" time="2017.778">
    <testsuite name="tests/claritybitcoin/segwit_tx.test.ts" timestamp="2025-03-05T17:44:11.808Z" hostname="Mikes-MacBook-Pro.local" tests="1" failures="1" errors="0" skipped="0" time="0.014">
        <testcase classname="tests/claritybitcoin/segwit_tx.test.ts" name="clarity bitcoin &gt; segwit transaction" time="0.014">
            <failure message="expected { type: 8, value: { type: 1, …(1) } } to deeply equal { type: 8, …(1) }" type="AssertionError">
AssertionError: expected { type: 8, value: { type: 1, …(1) } } to deeply equal { type: 8, …(1) }

- Expected
+ Received

  Object {
    &quot;type&quot;: 8,
    &quot;value&quot;: Object {
      &quot;type&quot;: 1,
-     &quot;value&quot;: 2200n,
+     &quot;value&quot;: 8n,
    },
  }

 ❯ tests/claritybitcoin/segwit_tx.test.ts:59:27

            </failure>
        </testcase>
    </testsuite>
</testsuites>
